release-notes/3.0/preview/api-diff/preview8/.Net/3.0-preview8_System.Text.Encodings.Web.md

```diff
@@ -0,0 +1,14 @@
+# System.Text.Encodings.Web
+
+``` diff
+ namespace System.Text.Encodings.Web {
+     public abstract class JavaScriptEncoder : TextEncoder {
++        public static JavaScriptEncoder UnsafeRelaxedJsonEscaping { get; }
+     }
+     public abstract class TextEncoder {
++        public virtual OperationStatus EncodeUtf8(ReadOnlySpan<byte> utf8Source, Span<byte> utf8Destination, out int bytesConsumed, out int bytesWritten, bool isFinalBlock = true);
++        public virtual int FindFirstCharacterToEncodeUtf8(ReadOnlySpan<byte> utf8Text);
```

Comment:
>FindFirstCharacterToEncodeUtf8 [](start = 28, length = 30)

This API has [EditorBrowsable(EditorBrowsableState.Never)] attribute, should we include this attribute? or we don't do that?

Reply:
I checked and I didn't see any precedent for this attribute.

Reply:
I would recommend to add the attribute to show the intend of this API. but I am not feeling strongly about it.

Reply:
We don't do this elsewhere.